ngraph/test/pass/serialization/rt_info_serialization.cpp

```diff
@@ -4,13 +4,12 @@
 
```

Comment:
in file `serialization/rt_info_deserialization.cpp` we use IE Core to read the IR. You can copy logic from here which works with IR frontend https://github.com/openvinotoolkit/openvino/pull/7853